src/main/java/com/easypost/model/CreditCard.java

```diff
@@ -0,0 +1,78 @@
+package com.easypost.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.easypost.EasyPost;
+import com.easypost.exception.EasyPostException;
+
+public class CreditCard extends BaseCreditCard {
+    public enum Priority {
+        PRIMARY,
+        SECONDARY
+    }
+
+    /**
+     * Fund a credit card.
+     *
+     * @param primaryOrSecondary primary or secondary payment method.
+     * @param amount amount to fund.
+     * @return CreditCardFund object.
+     * @throws EasyPostException when the request fails.
+     */
+    public static CreditCardFund fund(Priority primaryOrSecondary, int amount) throws EasyPostException {
+        return fund(primaryOrSecondary, amount, null);
+    }
+
+    /**
+     * Fund a credit card.
+     *
+     * @param primaryOrSecondary primary or secondary payment method.
+     * @param amount amount to fund.
+     * @param apiKey API key to use in request (overrides default API key).
+     * @return CreditCardFund object.
+     * @throws EasyPostException when the request fails.
+     */
+    public static CreditCardFund fund(Priority primaryOrSecondary, int amount, String apiKey) throws EasyPostException {
+        PaymentMethod paymentMethods = PaymentMethod.all();
+        String cardID = null;
+
+        if (primaryOrSecondary == Priority.PRIMARY) {
+            cardID = paymentMethods.getPrimaryPaymentMethod().getId();
+        } else {
```

Comment:
Let's be more explicit here and do an else if and check if it matches `SECONDARY`

Reply:
Or even better, a switch statement